### PR TITLE
Update with tab completion for kubectl

### PR DESCRIPTION
**Pull request by dsalamancaMS**
Update with tab completion for kubectl

### CODE REVIEW

**Comment by tokiwinter** [comment]
You could just `. ~/.bashrc` after adding the line to save spawning another shell, or at least, `exec` it?

**Comment by dsalamancaMS** [comment]
Thanks @tokiwinter I didnt think of that. Reloading bashrc makes more sense than just spawning a child shell

File: docs/02-client-tools.md [hunk 1]
```diff
@@ -99,6 +99,13 @@ chmod +x kubectl
 ```
 sudo mv kubectl /usr/local/bin/
 ```
+The following steps will enable shell tab completion on kubectl while using bash:
+
+```
+echo "source <(kubectl completion bash)" >> ~/.bashrc && .~/.bashrc
```
Comment (alfonsocantos) [inline docs/02-client-tools.md]: `.~/.bashrc` is it that correct (no space between dot a ~)? or it should be  `. ~/.bashrc` (with space between dot and ~)

Comment (dsalamancaMS) [reply docs/02-client-tools.md]: totally missed the space

**Comment by kelseyhightower** [comment]
While this is useful I'm really want to keep things simple and focus on the absolute essentials.

**Pull request closed without merging**